packages/pwa-kit-dev/src/utils/script-utils.ts

```diff
@@ -255,9 +278,27 @@ export const createBundle = async ({
                     })
             )
             .then(async () => {
-                const {dependencies = {}, devDependencies = {}} = await getProjectPkg()
+                const {
+                    dependencies = {},
+                    devDependencies = {},
+                    ccExtensibility = {extends: '', overridesDir: ''}
+                } = await getProjectPkg()
+                const extendsTemplate = 'node_modules/' + ccExtensibility.extends
+
+                let cc_overrides: string[] = []
+                if (ccExtensibility.overridesDir) {
+                    const overrides_files = await walkDir(
+                        ccExtensibility.overridesDir,
+                        ccExtensibility.overridesDir
+                    )
+                    const extends_files = await walkDir(extendsTemplate, extendsTemplate)
+                    cc_overrides = Array.from(overrides_files).filter((item) =>
+                        extends_files.has(item)
```

Comment:
If all we need `extends_files` for is for pairing with an `overrides_file`, then we don't actually need to walk the whole `extendsTemplate` directory, which likely contains many more files than the overrides directory. It should be sufficient to do `overrides_files.filter((item) => fs.existsSync(path.join(extendsTemplate, item)))`.